wp-content/themes/access/src/js/feedback.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-env browser */
+
+import Forms from '@nycopportunity/patterns-framework/src/utilities/forms/forms';
+import Modal from 'pattern-modal/src/modal';
+import serialize from 'for-cerial';
+
+(() => {
+  'use strict';
+  /**
+   * Pass the DOM element to the form.
+   */
+  const form = document.getElementById('feedback-form');
+  const Form = new Forms(form);
+  const modal = new Modal();
+
+  /**
+   * A set of strings to override the
+   */
+  Form.strings = {
+    'VALID_REQUIRED': 'This is required', // A generic message for required
+    // inputs that are missing values.
+    'VALID_{{ TYPE }}_INVALID': 'Invalid' // A validation message for a specific
+    // type. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#%3Cinput%3E_types
+    // for available types.
+  };
+
+  /**
+   * This function automatically watches inputs within the form and displays
+   * error messages on the blur event for each input.
+   */
+  Form.watch();
+
+  /**
+   * The submit function for the form.
+   */
+  Form.submit = (event) => {
+    event.preventDefault();
+
+    // To send the data with the application/x-www-form-urlencoded header
+    // we need to use URLSearchParams(); instead of FormData(); which uses
+    // multipart/form-data
+    let data = serialize(Form.FORM, {hash: true});
+    let formData = new URLSearchParams();
+
+    Object.keys(data).map(k => {
+      formData.append(k, data[k]);
+    });
+
+    let html = document.querySelector('html');
+
+    if (html.hasAttribute('lang'))
+      formData.append('lang', html.getAttribute('lang'));
+
+    fetch(Form.FORM.getAttribute('action'), {
+      method: Form.FORM.getAttribute('method'),
+      body: formData
+    })
+      .then(response => {
+        if (!response.ok) {
+          throw new Error('Network response was not ok');
+        }
```

Comment:
Display messaging "Something went wrong. Please try again later." similar to the text message module 

![image](https://user-images.githubusercontent.com/3826974/116465116-cc0e7080-a83a-11eb-9f2b-12ffb1870a09.png)

Reply:
This method will get the same localized string;

```
import localize from 'utilities/localize/localize';

localize('SHARE_FORM_SERVER') // < will return the localized string 
```